### ISSUE TITLE
Spaces are replaced with colons when transporting the environment variables into WSL

### ISSUE THREAD

**Issue by Satinelamp** [MicrosoftDocs/WSL#1057]
I am using windows-subsystem-for-Linux, and I choose Ubuntu. But I can't call the program installed on the Windows OS, so I check the environment variable list in WSL, it seems that the spaces are replaced with colons when transporting the environment variables into WSL.<br>
Considering that in WSL, the environment variables list is a colon-delimited list. In Win32, it is a semicolon-delimited list. <br>
I think this the reason why I can't call the program installed on the windows OS. So I checked another laptop, this phenomenon doesn't exist and everything works fine. 

My question is:

 1. Why does this problem happen?
 2. How could I fix it?

![6d2Ny](https://user-images.githubusercontent.com/62006565/104163641-b31bb400-5431-11eb-86b5-05e087c10d62.png)
![wIYfe](https://user-images.githubusercontent.com/62006565/104163655-b878fe80-5431-11eb-894b-d992f37bbaaa.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: deab370f-a7a1-ed3f-89ae-22194fd8fb78
* Version Independent ID: 9f406c77-415c-80db-d596-0a7fa4ea2ba9
* Content: [Manage Linux Distributions](https://docs.microsoft.com/en-us/windows/wsl/wsl-config)
* Content Source: [WSL/wsl-config.md](https://github.com/MicrosoftDocs/wsl/blob/master/WSL/wsl-config.md)
* Product: **dev-environment**
* Technology: **windows-subsystem-for-linux**
* GitHub Login: @craigloewen-msft
* Microsoft Alias: **crloewen**

**Comment by pchang388** [comment]
I did a fresh install on Windows 11 of wsl2 and Ubuntu 22.04.2 LTS distro (from Microsoft store) and ran into this exact same issue. I noticed that this only happened with I selected a versioned kernel from the distro instead of the generic "ubuntu" distro.

I am unsure why this happens but appears to be an issue with this specific distro and how it handles windows pathing in the "Ubuntu 22.04.2 LTS" offering. When I installed "Ubuntu" distro instead, it was already installed as `VERSION="22.04.3 LTS (Jammy Jellyfish)"` which means it is an issue with 22.04.2 distro or an issue with with that specific distro on the marketplace.

I would try just using the generic "ubuntu" (defaults to latest LTS) distro in the store and ditching the versioned one
